src/header/css/header-menus.pcss

```diff
@@ -236,6 +236,84 @@
 /* Global Header Menus: Submenu
 /* ========================================================================== */
 
+.esri-header-menus-submenu  {
+	& .esri-header-menus-structured-menu--wrapper {
+		& .esri-header-menus-submenu--left-col {
+			float: left;
```

Comment:
Do you need to nest this class under the other two? Generally the point of BEM is to lower the specificity of the selectors. If possible, I would recommend just writing `.esri-header-menus-submenu--left-col {` at the root level of depth. Don't nest unless you have to.

Also, you can use the logical properties plugin to do floats if you want:

```
float: inline-start;
```

This will evaluate to `left` in left to right languages and `right` in right to left languages.

Reply:
@paulcpederson the reason for the specificity is to only target those two column layouts.. By removing this I am having some clashing with other classes.

Reply:
the BEM way to do this would be to use a modifier class on the `esri-header-menus-submenu--left-col`, something like `esri-header-menus-submenu--left-col__structured` and then target it directly with just the properties that have changed.